package.json

```diff
@@ -5,7 +5,7 @@
   "style": "dist/accessible-autocomplete.min.css",
   "description": "An autocomplete component, built to be accessible.",
   "repository": "alphagov/accessible-autocomplete",
-  "author": "Theodor Vararu <theodor.vararu@digital.cabinet-office.gov.uk>",
+  "author": "Government Digital Service developers (https://gds.blog.gov.uk/)",
```

Comment:
Could I suggest "Government Digital Service" and link [here](https://www.gov.uk/government/organisations/government-digital-service) as that matches what we put in the footer of GOV.UK?